vmdb/spec/models/miq_schedule_spec.rb

```diff
@@ -684,7 +684,12 @@
   end
 
   context "#verify_file_depot" do
-    let(:params) { {:uri_prefix => "ftp", :uri => "ftp://ftp.example.com", :name => "Test Backup Depot", :username => "user", :password => "password"} }
```

Comment:
@brandondunne If you recall, the test was failing earlier only on Travis and was working fine locally, thats when I had made a few changes in the test ... the main goal was to identify the root cause of why it was failing only on Travis.
I have now restored all those changes, since we have identified the root cause.